jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/transform/node/incoming/TccGeneratorNode.kt

```diff
@@ -101,11 +125,39 @@ class TccGeneratorNode(
                 // TODO: Chrome does something more advanced, keeping older sequences to replay on packet reordering.
                 packetArrivalTimes.clear()
             }
-            if (windowStartSeq == -1 || tccSeqNum < windowStartSeq) {
-                windowStartSeq = tccSeqNum
-            }
 
-            timestamp?.run { packetArrivalTimes.putIfAbsent(tccSeqNum, timestamp) }
+            timestamp?.run {
+                if (packetArrivalTimes.isEmpty()) {
+                    lossListeners.forEach {
+                        it.packetReceived(false)
+                    }
+                } else {
+                    val oldMax = packetArrivalTimes.lastKey()
+                    if (tccSeqNum > oldMax) {
+                        val numLost = tccSeqNum - oldMax + 1
```

Comment:
I think this should be `-1`, right now it counts 2 missing packets for every packet that arrives in order :)